react-native-git-upgrade/package.json

```diff
@@ -19,6 +19,7 @@
     "babel-preset-es2015-node": "^6.1.1",
     "babel-preset-stage-3": "^6.17.0",
     "babel-register": "^6.18.0",
+    "minimist": "^1.2.0",
```

Comment:
minimist is required in [`index.js:12`](https://github.com/facebook/react-native/blob/master/react-native-git-upgrade/index.js#L12)